core/dispatcher/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -332,27 +337,49 @@ class Invoker(
      *
      * Note: Updates the container's log cursor to indicate consumption of log.
      */
-    private def getContainerLogs(con: WhiskContainer, tries: Int = LogRetryCount)(implicit transid: TransactionId): JsArray = {
+    private def getContainerLogs(con: WhiskContainer, limits: ActionLimits, tries: Int = LogRetryCount)(implicit transid: TransactionId): JsArray = {
         val size = con.getLogSize(runningInContainer)
+        val maxSize = limits.logs.map { _.megabytes }.getOrElse(Int.MaxValue)
```

Comment:
really? `getOrElse(default log limit makes more sense to me)`. this grandfathers actions to unbounded log length, is this intended?